src/View/InstallQuickInput.ts

```diff
@@ -59,62 +58,60 @@ class InstallQuickInput {
 
   isUpdateBackend: boolean = false;
 
-  constructor() {}
-
-  getToolchainEnv(): ToolchainEnv {
+  public getToolchainEnv(): ToolchainEnv {
     if (this.error !== undefined || this.toolchainEnv === undefined) {
       throw new Error('wrong calling');
     }
     return this.toolchainEnv;
   }
 
-  getToolchainType(): string {
+  public getToolchainType(): string {
     if (this.error !== undefined || this.toolchainType === undefined) {
       throw new Error('wrong calling');
     }
     return this.toolchainType;
   }
 
-  getToolchain(): Toolchain {
+  public getToolchain(): Toolchain {
     if (this.error !== undefined || this.toolchain === undefined) {
       throw new Error('wrong calling');
     }
     return this.toolchain;
   }
 
-  getVersion(): string {
+  public getVersion(): string {
     if (this.error !== undefined || this.version === undefined) {
       throw new Error('wrong calling');
     }
     return this.version;
   }
 
-  getError(): string|undefined {
+  public getError(): string|undefined {
     return this.error;
   }
 
-  getQuickPickItems(items: string[]): vscode.QuickPickItem[] {
+  public getQuickPickItems(items: string[]): vscode.QuickPickItem[] {
     return items.map(label => ({label}));
   }
 
-  getAllToolchainEnvNames(): string[] {
+  public getAllToolchainEnvNames(): string[] {
```

Comment:
FYI, if they are private functions, then it's not necessary to be cover them separately in unit test as long as they can be covered by other public functions. 

I think some of those functions have been used a private because they don't need to be used outside. In terms of encapsulation, I think you can keep some of them as private rather then exposing them..

Reply:
Thank you for information. :) This function is originally a public method. I just added the keyword explicitly. :)